components/base/sensorcontrolled/sensorcontrolled.go

```diff
@@ -251,12 +311,24 @@ func (sb *sensorBase) SetPower(
 ) error {
 	sb.opMgr.CancelRunning(ctx)
 	sb.setPolling(false)
+	if sb.angularVelocityLoop != nil {
+		sb.angularVelocityLoop.Stop()
```

Comment:
[q] Does this make the loop nil? What is the difference between non-nil and started?

Reply:
no, but unclear if you can restart it after stopping

Reply:
There is now one star and one stop - we will deal with the loop activity in the SetState and State functions